website/pages/en/chain-integration-overview.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Chain Integration Process Overview
+---
+
+A transparent and governance-based integration process was designed for blockchain teams seeking integration with The Graph protocol. It is a 3-phase process, as summarised below.
+
+## Stage 1. Technical Integration
+
+- Teams work on a Graph Node integration. [Here's how](/new-chain-integration/).
+- Teams initiate the protocol integration process by creating a Forum thread [here](https://forum.thegraph.com/c/governance-gips/new-chain-support/71) (New Data Sources sub-category under Governance & GIPs). Using the default Forum template is mandatory.
+
+## Stage 2. Integration Validation
+
+- Teams collaborate with the Graph Foundation and operators of GUIs and network gateways, such as [Subgraph Studio](https://thegraph.com/studio/), to ensure a smooth integration process. This involves providing the necessary backend infrastructure, such as the integrating chain's JSON RPC or Firehose endpoints. Teams wanting to avoid self-hosting such infrastructure can leverage The Graph's community of node operators (Indexers) to do so, which the Foundation can help with.
+- Graph Indexers test the integration on The Graph's testnet.
+- Core developers and Indexers monitor stability, performance, and data determinism.
+
+## Stage 3. Mainnet Integration
+
+- Teams propose mainnet integration by submitting a Graph Improvement Proposal (GIP) and initiating a pull request (PR) on the [feature support matrix](https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md) (more details on the link).
+- The Graph Council reviews the request and approves mainnet support, providing a successful Stage 2 and positive community feedback.
+
+---
+
+If the process looks daunting, don't worry! The Graph Foundation is committed to supporting integrators by fostering collaboration, offering essential information, and guiding them through various stages, including navigating governance processes such as Graph Improvement Proposals (GIPs) and pull requests. If you have questions, please reach out to [info@thegraph.foundation](mailto:info@thegraph.foundation) or through Discord (either Pedro, The Graph Foundation member, IndexerDAO, or other core developers).
+
+---
+
+## Frequently Asked Questions
+
+**1. How does this relate to the [World of Data Services GIP](https://forum.thegraph.com/t/gip-0042-a-world-of-data-services/3761)?**
```

Comment:
@MichaelMacaulay – I know you merged this already, but I feel like these should be headings?